lib_xua/api/xua_conf_default.h

```diff
@@ -717,7 +716,7 @@
  * @brief Device firmware version number in Binary Coded Decimal format: 0xJJMN where JJ: major, M: minor, N: sub-minor version number.
  */
 #ifndef BCD_DEVICE_J
-#define BCD_DEVICE_J             (1)
+#define BCD_DEVICE_J             (4)
```

Comment:
I didn't know that this was supposed to track lib_xua's version number. Also, shouldn't it be 5.0.0 then? @danielpieczko and I were also discussing the other day that we need a test for this. In sw_usb_audio as well

Reply:
Yeah it needs to move to 5.0.0 when we make the 5.0.0 release

Reply:
I'm adding the test as part of v5.0.0 pre release PR